bson-kotlin/build.gradle.kts

```diff
@@ -111,12 +111,11 @@ spotbugs { showProgress.set(true) }
 // ===========================
 //     Test Configuration
 // ===========================
-tasks.create("kotlinCheck") {
```

Comment:
I think these can be deleted. Running `--dry-run` with gradle shows you the tasks to be run:

```
./gradlew bson-kotlin:check --dry-run | grep "bson-kotlin"
:bson-kotlin:compileKotlin SKIPPED
:bson-kotlin:compileJava SKIPPED
:bson-kotlin:compileGroovy SKIPPED
:bson-kotlin:processResources SKIPPED
:bson-kotlin:classes SKIPPED
:bson-kotlin:jar SKIPPED
:bson-kotlin:inspectClassesForKotlinIC SKIPPED
:bson-kotlin:checkstyleMain SKIPPED
:bson-kotlinx:compileKotlin SKIPPED
:bson-kotlinx:compileJava SKIPPED
:bson-kotlinx:compileGroovy SKIPPED
:bson-kotlinx:processResources SKIPPED
:bson-kotlinx:classes SKIPPED
:bson-kotlinx:jar SKIPPED
:bson-kotlinx:inspectClassesForKotlinIC SKIPPED
:bson-kotlin:compileTestKotlin SKIPPED
:bson-kotlin:compileTestJava SKIPPED
:bson-kotlin:compileTestGroovy SKIPPED
:bson-kotlin:processTestResources SKIPPED
:bson-kotlin:testClasses SKIPPED
:bson-kotlin:checkstyleTest SKIPPED
:bson-kotlin:codenarcMain SKIPPED
:bson-kotlin:codenarcTest SKIPPED
:bson-kotlin:detekt SKIPPED
:bson-kotlin:spotbugsMain SKIPPED
:bson-kotlin:spotbugsTest SKIPPED
:bson-kotlin:spotlessExtraneous SKIPPED
:bson-kotlin:spotlessExtraneousApply SKIPPED
:bson-kotlin:spotlessKotlin SKIPPED
:bson-kotlin:spotlessKotlinApply SKIPPED
:bson-kotlin:spotlessKotlinGradle SKIPPED
:bson-kotlin:spotlessKotlinGradleApply SKIPPED
:bson-kotlin:spotlessApply SKIPPED
:bson-kotlin:spotlessExtraneousCheck SKIPPED
:bson-kotlin:spotlessKotlinCheck SKIPPED
:bson-kotlin:spotlessKotlinGradleCheck SKIPPED
:bson-kotlin:spotlessCheck SKIPPED
:bson-kotlin:test SKIPPED
:bson-kotlin:check SKIPPED
```

Reply:
Good point 👍